access-common/pom.xml

```diff
@@ -13,19 +13,16 @@
 		<dependency>
 			<groupId>org.slf4j</groupId>
 			<artifactId>slf4j-api</artifactId>
-			<version>1.6.1</version>
```

Comment:
why was (only) the version removed here?

Reply:
slf4j is being retained, but the version is being set at the root pom instead of in each module pom